client/src/BookingTimeRangeSelector.tsx

```diff
@@ -0,0 +1,44 @@
+import React, { useState, FC } from "react";
+import { Typography, Card, CardContent, TextField } from "@mui/material";
+import AdapterDateFns from "@mui/lab/AdapterDateFns";
+import { LocalizationProvider, StaticDatePicker, TimePicker } from "@mui/lab";
```

Comment:
Is it not possible to add the import AdapterDateFns to from this line?

Reply:
it gave an error that there is no field called AdapterDateFns in @mui/lab when i tried to import it from that line